### DISCUSSION TITLE
💫Mini-Stronies -- the Stone Soup awards 2024💫

### DISCUSSION THREAD

**Discussion by jmbarr** [dstl/Stone-Soup#944] [General]
Following the runaway success of last year's event, the annual Stone Soup awards -- the Mini-Stronies, are back for 2024 and they promise to be the best ever! 

This year we need your help in two important respects:
1. Nominate. We're looking for candidates for the award of (i) best contribution, (ii) most improved class, (iii) small code contribution, and (iv) best documentation. If there's been a significant contribution to Stone Soup that you think should be acknowledged, please let us know
2. We're looking to incorporate new categories in the awards. Perhaps there's an underrepresented area we should acknowledge, or applications we should encourage, or maybe there's an opportunity to get competitive. Just drop us a line.

Polls will appear here round about mid-March. Winners announced at OSTEWG in July. Good luck!

**Comment by sdhiscocks** [comment]
As an improvement to class (or guess multiple classes on same base in the case), #817 added 3 new resampling methods, where as before there was only one.

**Comment by sdhiscocks** [comment]
#760 documentation of Reinforcement Learning was a great example of using Stone Soup with other libraries to do sensor management.

**Comment by jmbarr** [comment]
#974 #975 #976
